sdk/highlight-cloudflare/src/sdk.ts

```diff
@@ -1,15 +1,15 @@
 /* Required to patch missing performance API in Cloudflare Workers. */
-import 'opentelemetry-sdk-workers/performance'
+import '@highlight-run/opentelemetry-sdk-workers/performance'
 
-import { OTLPProtoLogExporter } from 'opentelemetry-sdk-workers/exporters/OTLPProtoLogExporter'
-import { OTLPProtoTraceExporter } from 'opentelemetry-sdk-workers/exporters/OTLPProtoTraceExporter'
+import { OTLPProtoLogExporter } from '@highlight-run/opentelemetry-sdk-workers/exporters/OTLPProtoLogExporter'
+import { OTLPProtoTraceExporter } from '@highlight-run/opentelemetry-sdk-workers/exporters/OTLPProtoTraceExporter'
 import { Resource } from '@opentelemetry/resources'
 import type { ResourceAttributes } from '@opentelemetry/resources/build/src/types'
-import { WorkersSDK } from 'opentelemetry-sdk-workers'
+import { WorkersSDK } from '@highlight-run/opentelemetry-sdk-workers'
 
 const HIGHLIGHT_PROJECT_ENV = 'HIGHLIGHT_PROJECT_ID'
 const HIGHLIGHT_REQUEST_HEADER = 'X-Highlight-Request'
-const HIGHLIGHT_OTLP_BASE = 'https://otel.highlight.io'
+const HIGHLIGHT_OTLP_BASE = 'https://otel.highlight.io:4318'
```

Comment:
The Cloudflare SDK was broken if the user didn't pass in an OTLP endpoint. This repairs the default.